client/post-editor/editor-gutenberg-opt-in-sidebar/index.jsx

```diff
@@ -5,7 +5,7 @@
  */
 
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { PureComponent } from 'react';
```

Comment:
we can use `Component` here - fun fact! `connect()` automatically provides the same behavior as `PureComponent` unless we explicitly opt-out by passing `pure: false` in its options.

using `Component` keeps things a little more consistent and technically, though it won't matter, it's faster without because using `PureComponent` inside of `connect` means we run the same `shouldComponentUpdate` check every time the component _does_ need to re-render

Reply:
Ah! Cool, thanks <3